src/index.css

```diff
@@ -3,3 +3,11 @@ body {
   padding: 0;
   font-family: sans-serif;
 }
+
+.hidden{
+  display: none;
+}
+
+.fn-submit-name{
+  cursor: pointer;
```

Comment:
It looks like this class is being applied to elements which have the `.btn` class from Bootstrap. These should already get `cursor: pointer` styles, so this is unnecessary.

Reply:
It doesn't show the pointer when I press the button that's why I added it.

Reply:
Thank you for your comments

Reply:
@LouiaR good work on the last commit, tidying the code, better indentation, remove unnecessary comments and renaming for clarity.